### DISCUSSION TITLE
Enhancing Budget Management in FireFlyIII: Introducing Target Tracking

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#8969] [General]
Hello,

I'm currently exploring the FireFlyIII app, and I must say, it's an impressive tool that provides clear visibility into my expenses. The Budget functionality is quite effective, but I believe it could benefit from additional features to help prevent overspending.

One idea I have is to include a "target" column alongside the "budgeted" column on the /budget page. This way, we can easily monitor whether we're staying within our target, even if we adjust the budget throughout the month.

Best regards,

**Comment by JC5** [comment]
I imagine the budget **is** the target.
